admin-dev/themes/new-theme/template/layout.tpl

```diff
@@ -12,6 +12,8 @@
 <header>
   <nav class="main-header">
 
+    <button class="btn btn-primary-reverse onclick btn-lg unbind ajax-spinner"></button>
```

Comment:
The UI Kit will be updated because a lot of things do not match the mockups. So, for now, I'd say it's okay to have a button.
You can follow the work on the dedicated branch here: https://github.com/PrestaShop/prestashop-ui-kit/tree/develop
I'm also preparing a branch (soon a PR) to integrate this new UI Kit into PrestaShop.